SimG4CMS/PPS/interface/PPSDiamondG4Hit.h

```diff
@@ -0,0 +1,122 @@
+///////////////////////////////////////////////////////////////////////////////
+// Author
+// Seyed Mohsen Etesami setesami@cern.ch
+// Feb 2016
+///////////////////////////////////////////////////////////////////////////////
+#ifndef PPS_PPSDiamondG4Hit_h
+#define PPS_PPSDiamondG4Hit_h
+
+#include "G4VHit.hh"
+#include "G4ThreeVector.hh"
+#include <boost/cstdint.hpp>
```

Comment:
@mundim , boost dependence here is not needed , you may use #include <cstdint>

Reply:
sorry about it, this scaped a previous search to remove all boost dependency.

Reply:
Done